be/src/runtime/tablets_channel.cpp

```diff
@@ -393,7 +394,7 @@ Status TabletsChannel::reduce_mem_usage_async(const std::set<int64_t>& flush_tab
 }
 
 Status TabletsChannel::wait_mem_usage_reduced(int64_t tablet_id) {
-    vectorized::DeltaWriter* vectorized_writer = nullptr;
+    std::shared_ptr<vectorized::DeltaWriter> vectorized_writer;
```

Comment:
std::unique_ptr Can be used here ?

Reply:
`vectorized_writer` is a reference to element in `_vectorized_tablet_writers`, we can't use `unique_ptr`.